feat: Implement ykush auxiliary (#225)
eclipse/kiso-testing: docs/modules_usage/ykush_control.rst

.. _ykush_aux:

Controlling an Yepkit USB hub

> ===============================

Reviewer: ```suggestion
=============================
```

Author: Done

> The ykush auxiliary offers commands to control an Yepkit usb hub.
> Ports can be switched on and off individually.

Reviewer 2: ```suggestion
The ykush auxiliary offers commands to control an Yepkit USB hub, 
allowing to switch the USB ports on and off individually.
```

> 

Reviewer: maybe add a link to the product
https://www.yepkit.com/products/ykush

Author: Done
